llama-index-integrations/readers/llama-index-readers-mediawiki/llama_index/readers/mediawiki/base.py

```
@@ -0,0 +1,376 @@
+        reader.login("username", "password")
+        docs = reader.load_data()
+
+    Implements BasePydanticReader (for serialization / LlamaHub compatibility).
+
+    """
+
+    FILTERREDIR_NONREDIRECTS: str = "nonredirects"
+    FILTERREDIR_ALL: str = "all"
+
+    is_remote: bool = True
+    model_config = {"arbitrary_types_allowed": True}
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)